index-import.cpp

```diff
@@ -254,10 +257,10 @@ static std::string normalizePath(StringRef Path) {
 
 static bool remapIndex(const Remapper &remapper,
                        const std::string &InputIndexPath,
-                       const std::string &outputIndexPath) {
+                       const std::string &outputIndexPath, std::ostream *outs) {
```

Comment:
looks like this should be on the next line?

Reply:
It does, but not by clang-format's llvm settings I guess.